code/game/objects/obj_defense.dm

```diff
@@ -204,6 +204,7 @@ GLOBAL_DATUM_INIT(acid_overlay, /mutable_appearance, mutable_appearance('icons/e
 		SSfire_burning.processing[src] = src
 		add_overlay(GLOB.fire_overlay, TRUE)
 		return 1
+	. = ..()
```

Comment:
can you make this more explicit in your pr as this may have side effects elsewhere

Reply:
uh what should i say? because i'm not sure what this change can cause

Reply:
```suggestion
	return ..()
```

Reply:
if i add it to the start of the proc it causes runtime errors

Reply:
```suggestion
	return ..()
```
I don't like the whole logic of this, but it's a fix, so whatever.

Reply:
committed